doc/kubernetes/modules/ROOT/pages/running/concepts/infinispan-cache-config-xml.adoc

```diff
@@ -0,0 +1,72 @@
+
+== Concepts
+
+When configuring remote cache in Keycloak, it's essential to understand the specific needs of your deployment, in other words, whether you are focusing on high availability, fault tolerance, scalability, or a combination of these. See https://www.keycloak.org/server/caching[Configuring distributed caches in Keycloak] for more details.
+
+Below is an example `<distributed-cache>` XML block for `sessions` named cache, we can configure it for other named caches in a similar way.
+
+[source,xml]
+----
+<distributed-cache name="sessions" owners="2" statistics="true">
+            <expiration lifespan="-1"/>
+            <persistence passivation="false">
+                <remote-store xmlns="urn:infinispan:config:store:remote:14.0"
+                              cache="sessions"
+                              raw-values="true"
+                              shared="true"
+                              segmented="false">
+                    <remote-server host="${env.KC_REMOTE_STORE_HOST}"
+                                   port="${env.KC_REMOTE_STORE_PORT}"/>
+                    <connection-pool max-active="16"
+                                     exhausted-action="CREATE_NEW"/>
+                    <security>
+                        <authentication server-name="infinispan">
+                            <digest username="${env.KC_REMOTE_STORE_USERNAME}"
+                                    password="${env.KC_REMOTE_STORE_PASSWORD}"
+                                    realm="default"/>
+                        </authentication>
+                        <encryption protocol="TLSv1.3"
+                                    sni-hostname="${env.KC_REMOTE_STORE_HOST}">
+                            <truststore filename="/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt"
+                                        type="pem"/>
+                        </encryption>
+                    </security>
+                </remote-store>
+            </persistence>
+            <state-transfer enabled="false"/>
+        </distributed-cache>
+----
+
+
+Properly tuned cache settings can significantly improve the performance and reliability of your Keycloak application. Following are some key concepts in configuring a remote cache store.
+
+=== Cache containers
+Within the XML, you will find a `<cache-container>` tag, which defines the settings for caches that are not explicitly configured.
+
+=== Named Caches
+Inside the default cache container, there are `<local-cache>`, `<distributed-cache>` and other cache types. These named caches have specific roles, like `realms`, `users`, `sessions` etc.
+
+=== Owners
+In a distributed cache configuration `<distributed-cache>`, the `<owners>` XML attribute defines the number of cluster nodes that will store a copy of the cache entry. This is a critical attribute for fault tolerance and performance.
+
+=== Expiration
```

Comment:
We should not document this config option. We don't want users to change this as it can result in an inconsistent state when some entries are removed sooner than they should. Users should change it only when they really know what they are doing and this comment may sound like it is safe to change it as they want.